source/containers.tex

```diff
@@ -76,8 +76,8 @@
 \tcode{T}, \tcode{a} and \tcode{b}
 denote values of type \tcode{X}, \tcode{u}
```

Comment:
N3035 says:

> In Tables 91 and 92, `X` denotes a container class containing objects of type `T`, `a` and `b` denote values of type
`X`, `u` denotes an identifier, `r` denotes an lvalue or a const rvalue of type `X`, and `rv` denotes a non-const rvalue
of type `X`.

By N3090 the distinction had been lost.